packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts

```diff
@@ -78,6 +71,12 @@ export class OrgList implements vscode.Disposable {
     const authList = [];
     const today = new Date();
     for (const orgAuth of orgAuthorizations) {
+      if (orgAuth.error) {
+        console.warn(
+          `Org Auth for username: ${orgAuth.username} has an error: ${orgAuth.error}`
+        );
+        continue;
+      }
```

Comment:
Confirmed that this is still needed, even with the additional call to StateAggregator.clearInstance() in workspaceContextUtil.ts.  

Even after making that call, listAllAuthorizations(), Will still sometimes return the stale cached auth with an error.

Reply:
can we add a code comment about why this is necessary?